plantuml-parser-core/src/test/java/com/shuzijun/plantumlparser/core/UmlGenerationTester.java

```diff
@@ -133,4 +110,11 @@ public static void main(String[] args) throws IOException {
         testFilesDefaultConstructor("ClassWOConstructorNoConstructorOutput.txt", "ClassWOConstructor.java", false);
         testFilesDefaultConstructor("ClassWithConstructorOutput.txt", "ClassWithConstructor.java", false);
     }
+
+    public static void main(String[] args) throws IOException {
+        // Test record
```

Comment:
You can drop these comments.

FYI, if you want to know my opinion son comments, see https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/